docs/commands/README.md

```diff
@@ -1,8 +1,3 @@
----
-sidebar_position: 3
-sidebar_label: Commands
----
-
```

Comment:
For these README files that act as intro text for a folder ("category") of docs (such as **Commands**), it's the `_category_.yaml` that has the active config that determines the position/label in the sidebar, so this stuff I've removed here was effectively a no-op.